certify-core/src/main/java/io/mosip/certify/core/dto/CredentialMetadata.java

```diff
@@ -17,7 +18,9 @@ public class CredentialMetadata {
     private String id;
     private String format;
     private String scope;
-    private List<String> proof_types_supported;
+    // proof_types_supported is a map from v13 & an array before
+    private Object proof_types_supported;
```

Comment:
We cannot declare as generic object type, can we have proper type in place. Also I would like to understand where we are using this value.

Reply:
Removed this as this isn't being used or written back to response.